src/wizard-settings/header-bidding-gam/order-popover.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { MenuItem, ExternalLink } from '@wordpress/components';
+import { moreVertical } from '@wordpress/icons';
+import { ESCAPE } from '@wordpress/keycodes';
+
+/**
+ * Newspack dependencies
+ */
+import { Button, Popover } from 'newspack-components';
+
+const OrderPopover = ( { disabled = false, isDraft, onEdit, onArchive, gamLink } ) => {
+	const [ isVisible, setIsVisible ] = useState( false );
+	const toggleVisible = () => setIsVisible( state => ! state );
+	return (
+		<>
+			<Button
+				isQuaternary
+				isSmall
+				disabled={ disabled }
+				className={ isVisible && 'popover-active' }
+				onClick={ toggleVisible }
+				icon={ moreVertical }
+				label={ __( 'More options', 'newspack' ) }
+				tooltipPosition="bottom center"
+			/>
+			{ ! disabled && isVisible && (
+				<Popover
+					position="bottom left"
+					onFocusOutside={ toggleVisible }
+					onKeyDown={ event => ESCAPE === event.keyCode && toggleVisible }
+				>
+					<MenuItem onClick={ toggleVisible } className="screen-reader-text">
+						{ __( 'Close Popover', 'newspack-ads' ) }
+					</MenuItem>
+					<MenuItem
+						onClick={ () => {
+							if ( typeof onEdit === 'function' ) onEdit();
+						} }
+						className="newspack-button"
+					>
+						{ __( 'Edit', 'newspack-ads' ) }
+					</MenuItem>
+					{ isDraft && (
+						<MenuItem
+							onClick={ () => {
+								if ( typeof onArchive === 'function' ) onArchive();
+							} }
+							className="newspack-button"
+						>
+							{ __( 'Archive', 'newspack-ads' ) }
```

Comment:
This is a fairly serious action, as it's irreversible on the GAM side. I'd suggest we pop up a browser warning asking if you're sure you want to archive this order. Source: I accidentally clicked "Archive" in the middle of testing before I got to that step. 😆

Reply:
You actually can unarchive an order through GAM, but I agree this action deserves a confirmation! Done in 11eb58a4a1eefeea9cec594e546b4bd6dbf8ddab